app/models/form/sales/subsections/setup.rb

```diff
@@ -17,6 +17,7 @@ def pages
       Form::Sales::Pages::DiscountedOwnershipType.new(nil, nil, self),
       Form::Sales::Pages::OutrightOwnershipType.new(nil, nil, self),
       Form::Sales::Pages::JointPurchase.new(nil, nil, self),
+      Form::Sales::Pages::PropertyNumberOfBedrooms.new(nil, nil, self),
```

Comment:
I think this is meant to be in Property information subsection, I think you'd need to create a new subsection, new section and add it to a form in the FormHandler

Reply:
It seems you've created the subsection but you still need to remove this line from here